ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -21,25 +21,28 @@ spec:
           working_dir: "https://github.com/ray-project/test_dag/archive/59c994656f55ff8e0e642efe63f380debc831931.zip"
         deployments:
           - name: Blocked
+            max_ongoing_requests: 1000
             autoscaling_config:
               metrics_interval_s: 0.2
               min_replicas: 1
               max_replicas: 14
               look_back_period_s: 2
               downscale_delay_s: 5
               upscale_delay_s: 2
-              target_num_ongoing_requests_per_replica: 1
+              target_ongoing_requests: 1
             graceful_shutdown_timeout_s: 5
-            max_concurrent_queries: 1000
             ray_actor_options:
               num_cpus: 0.5
       - name: signal
         import_path: autoscaling.signaling:app
         route_prefix: /signal
+        deployments:
+          - name: SignalDeployment
+            max_ongoing_requests: 1000
```

Comment:
Each `block` request will also send a request to `signal` replica, so `max_ongoing_requests` should more than 20 (number of block requests). The default value changes from 100 to 5 in Ray 2.32.0.